cmd/oras/internal/option/target.go

```diff
@@ -96,13 +97,21 @@ func (opts *Target) ApplyFlagsWithPrefix(fs *pflag.FlagSet, prefix, description
 
 // Parse gets target options from user input.
 func (opts *Target) Parse(cmd *cobra.Command) error {
+	if opts.IsOCILayout && opts.Path != "" {
+		return fmt.Errorf("flag %q is not supported with %q", "oci-layout-path", "oci-layout")
+	}
```

Comment:
Consider `CheckMutuallyExclusiveFlags`. Examples see
https://github.com/oras-project/oras/blob/97cb3766ad2055ddbf3cb5dd49c2725ae0b12148/cmd/oras/internal/option/remote.go#L160-L165

Reply:
Thanks, updated it and its corresponding test.